bot/exts/evergreen/avatar_modification/_effects.py

```diff
@@ -96,6 +96,18 @@ def eight_bitify_effect(image: Image.Image) -> Image.Image:
         image = image.resize((1024, 1024), resample=Image.NEAREST)
         return image.quantize()
 
+    @staticmethod
+    def reverse_effect(image: Image.Image) -> Image.Image:
```

Comment:
I feel like this is better named as `flip_effect`

Reply:
okay